src/i18n/en-events.ts

```diff
@@ -40,21 +40,21 @@ const eventsMessageDescriptors = {
   [`event.${ColonyAndExtensionsEvents.DomainMetadata}.name`]: `{initiator} changed teams's name from {oldName} to {domainName}`,
   [`event.${ColonyAndExtensionsEvents.DomainMetadata}.color`]: `{initiator} changed teams's color from {oldColor} to {domainColor}`,
   [`event.${ColonyAndExtensionsEvents.DomainMetadata}.fallback`]: `{initiator} changed this team, but values are the same`,
-  [`event.${ColonyExtendedActions.SafeTransactionInitiated}.rawTransaction`]: `{safeName} initiated a transaction with {isSafeTransactionRecipientUser, select, 
+  [`event.${ColonyAndExtensionsEvents.ArbitraryTransaction}.rawTransaction`]: `{safeName} initiated a transaction with {isSafeTransactionRecipientUser, select, 
     true {{safeTransactionRecipient}}
     other {{safeTransactionRawTransactionAddress}}
   }`, // @TODO Confirm w product
```

Comment:
Assuming these have been a-oked by product now?

Reply:
Yeah I think it's safe to remove these comments